packages/client/src/i18n/messages/views/formConfig.ts

```diff
@@ -230,7 +230,7 @@ const messagesToDefine: IFormConfigMessages = {
   },
 
   integrations: {
-    id: 'config.integrations.',
+    id: 'config.integrations',
     defaultMessage: 'Integrations',
     description: 'Title for Form Integrations Page'
   },
```

Comment:
Do we need this one as we have https://github.com/opencrvs/opencrvs-core/blob/50f037d54ca4caeaf00ebb4ffb524137e2796f38/packages/client/src/i18n/messages/views/navigation.ts#L107 this one?

Reply:
True it's a page title. I moved it to navigation as "formConfig" is related to another page in application config